shesha/supervisor/saxoPlusManager.py

```diff
@@ -315,7 +359,7 @@ def alive(self):
 
     first_stage = CompassSupervisor(config1, cacao=adopt)
     second_stage = CompassSupervisor(config2, cacao=adopt)
```

Comment:
I guess `first_stage` should be a `Stage1Supervisor` instead of a `CompassSupervisor`, right ?
Same for second_stage...

Reply:
Addressed by [d484b28](https://github.com/ANR-COMPASS/shesha/pull/44/commits/d484b284b183f9b2c04c235e5f67392db45019de)